src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php

```diff
@@ -625,6 +625,34 @@ protected function typeMacAddress(Fluent $column)
     }
 
     /**
+     * Get the SQL for a generated virtual column modifier.
+     *
+     * @param  \Illuminate\Database\Schema\Blueprint  $blueprint
+     * @param  \Illuminate\Support\Fluent  $column
+     * @return string|null
```

Comment:
it is possible that definition is changing in php 7.1, but for now, we're sticking with our original definition